app/models/user.rb

```diff
@@ -459,6 +466,14 @@ def set_projects_limit
     self.projects_limit = current_application_settings.default_projects_limit
   end
 
+  def set_current_streak
```

Comment:
But is that something that has to do with initializing a new user? Maybe we can make this more explicit?